src/Client.php

```diff
@@ -65,6 +65,8 @@ public function query() {
     /**
      * Executes the given Query and returns the result
      *
+     * This method should not be called manually.
```

Comment:
> That was just copied from the Ruby documentation

Oh, I see... yeah, we I think about it, that syntax do look like ruby :stuck_out_tongue: